deepchecks/base/display_pandas.py

```diff
@@ -12,30 +12,25 @@
 from typing import List, Union
 import warnings
 
-from IPython.core.display import display_html
 import pandas as pd
 from pandas.io.formats.style import Styler
 
-from . import check  # pylint: disable=unused-import
+from deepchecks.utils.strings import get_docs_summary, get_ellipsis
 
+from . import check  # pylint: disable=unused-import
```

Comment:
why that line is needed?

Reply:
no sure why it's showing that I added that, but it's to be able to use CheckResult typing without circular input - I think